internal/index/indexscanner/scanner_test.go

```diff
@@ -90,6 +95,63 @@ func TestReadPluginFile(t *testing.T) {
 	}
 }
 
+func TestReadReceiptFromFile(t *testing.T) {
+	type args struct {
+		receiptFilePath string
+		pluginName      string
+		indexName       string
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		{
+			name: "read receipt of plugin from default index without status",
```

Comment:
`from default index` part is superfluous.

this just a receipt without `status` field.